lib/matplotlib/tests/test_image.py

```diff
@@ -153,7 +153,7 @@ def test_imsave_color_alpha():
     data = random.rand(16, 16, 4)
 
     buff = io.BytesIO()
-    plt.imsave(buff, data)
+    plt.imsave(buff, data, origin='lower')
```

Comment:
probably should save this with both origins and check that you get the correct output in both cases.